services/botVerificationService.js

```diff
@@ -10,4 +10,14 @@ const verifyToken = (token) => {
   return jwt.verify(token, config.get("botToken.botPublicKey"), { algorithms: ["RS256"] });
 };
 
-module.exports = { verifyToken };
+/**
+ * Verifies if the JWT is valid. Throws error in case of signature error or expiry
+ *
+ * @param token {String} - JWT to be verified
+ * @return {Object} - Decode value of JWT
+ */
+const verifyCronJob = (token) => {
```

Comment:
here we are not checking  the expiration time of the token. This means that the function could allow a token to be used even if it has expired.

Reply:
The jwt.verify method will throw an error in case the token is expired, Thus I haven't explicitly checked for the expiry time.